### PR TITLE
Read cytokit csv

### PR DESCRIPTION
**Pull request by mccalluc**
This is preparatory to https://github.com/hubmapconsortium/portal-containers/issues/14

As discussed, the document is large enough that it might be useful to use Arrow.

### CODE REVIEW

**Comment by mccalluc** [comment]
(Sorry, need to update fixture...)

File: python/cytokit_reader.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import csv
+import argparse
+import json
+
+
+def round_conv(s):
+    # TODO: Truncating after decimal point might be slightly too aggressive?
```
Comment (manzt) [inline python/cytokit_reader.py:8]: I'll look into this with Arrow and compare file sizes.

File: scripts/process_cytokit.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -o errexit
+
+. ./scripts/utils.sh
+
+main() {
+    # Download and process data which describes cell locations,
+    # and gene expression levels. Multiple JSON output files are produced:
```
Comment (manzt) [inline scripts/process_cytokit.sh]: Will there be more output files than just cells.json?

**Review by manzt — APPROVED** [review]
This looks good to me. I don't think we've added Arrow things into vitessce yet. I will explore generating arrow and comparing file sizes if that's something we are running into. Feel free to merge this and then I'll take a look tomorrow with Arrow.

If you could comment on how large the output currently is, that would be helpful for tomorrow.

File: python/cytokit_reader.py (hunk 1 above)
Comment (manzt) [reply python/cytokit_reader.py:8]: Perhaps for the x and y coordinates, we should have some precision beyond the decimal point, but I'm not convinced for the gene "levels" this is necessary.

**Pull request merged**